content/500-support/100-help-articles/500-comparing-columns-through-raw-queries.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'Comparing columns of same table through raw queries'
+metaTitle: 'Comparing columns of same table through raw queries'
+metaDescription: 'Comparing columns of same table through raw queries'
```

Comment:
These two should probably be different.

Reply:
Updated the `metaDescription` content. Thanks for pointing that out.